azure-test/tests/azure_storage_account_local_user/main.tf

```diff
@@ -0,0 +1,70 @@
+provider "azurerm" {
```

Comment:
Would it make sense to move the contents of main.tf to variables.tf, in accordance with our integration test rules?